app/workers/principals/delete_job.rb

```diff
@@ -29,7 +29,7 @@
 #++
 
 class Principals::DeleteJob < ApplicationJob
-  queue_with_priority :low
+  queue_with_priority :below_normal
```

Comment:
Shouldn't this be `:above_normal` ?

Reply:
I've increased the priorities for jobs where users see a "loading screen" to above_normal. Deleting can stay below_normal, though, I say.